exp/tests/test_response_views.py

```diff
@@ -517,12 +614,15 @@ def test_get_appropriate_fields_in_json_downloads(self):
             self.assertNotEqual(data[0]["participant"][header], "")
         # # Check that the remaining headers ARE NOT present
         for header in self.child_labels_json_2:
-            self.assertEqual(data[0]["child"][header], "")
+            self.assertNotIn(header, data[0]["child"].keys())
```

Comment:
is `data[0]["participant"]` a dict? then you don't need `.keys()`; `"foo" in {"foo": "bar"}` works just fine. See: https://docs.python.org/3/tutorial/datastructures.html#dictionaries

Reply:
whoops, thanks